docs/pages/BlogPosts.json

```diff
@@ -1,6 +1,58 @@
 {
   "year": "2022",
   "digests": [
+    {
+      "week": "June 17, 2022",
+      "posts": [
+        {
+          "title": "Dark mode Figma variants, Phase 1",
+          "audience": ["Design"],
+          "imageSrc": "https://i.pinimg.com/originals/90/d6/86/90d6869063bd80e56fe0820eebf10aca.png",
+          "imageAltText": "A dark mode variant of Popover",
+          "content": "This has been a long time coming, but we've shipped our first phase of dark mode variants in our Gestalt libraries! This first phase has focused on core components for our [Gestalt for iOS library](https://www.figma.com/file/AHcKJDgb7E7YswlgW1wY8E/Gestalt-for-iOS) (e.g., Button, Avatar, Popover, etc.). Our next phase will be to offer dark mode for more complex components like Pin, Board, and other reps. We look forward to sharing many more updates _soon_."
```

Comment:
you may want to clarify that this is "Gestalt _Figma_ libraries"

Reply:
Fixed!